internal/snakegame/snake.go

```diff
@@ -24,7 +24,6 @@ func (sg *SnakeGame) initSnake() {
 }
 
 type snake struct {
-	isFoodEaten        bool
```

Comment:
This PR should not contain changes not related to config file.
But this time is ok, no changes are required.

Reply:
done